dom.js

```diff
@@ -608,6 +608,21 @@ Document.prototype = {
 		return rtv;
 	},
 	
+	getElementsByClassName: function(className) {
+		const pattern = new RegExp(`(^|\\s)${className}(\\s|$)`);
```

Comment:
Try merging in the latest master to your PR and re-running the tests to see if Node 10.x no longer complains about this line.